modules/installation-creating-aws-dns.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-dns-{context}"]
+= Creating DNS and load balancing components in AWS
+
+You must configure DNS and load balancing in Amazon Web Services (AWS) for your
+{product-title} cluster to use. The easiest way to create these components is
+to modify the provided CloudFormation template.
+
+If you need to, you can run the template multiple times within a single VPC.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. You might need to consult Red Hat support to ensure that
```

Comment:
How will they know?  Ideally, the test for UPI config is "I tried to launch a cluster on my infra, and it came up".  If that fails, it's on us to provide useful debugging/log-gathering tools to help identify the violated assumption and repair.  But that's going to be an incremental thing with no "now we've fixed it" moment.  Can we just reword this to say "try it and see, and fall back to support with your tool-gathered logs if you get stuck"?